templates/crate/details.html

```diff
@@ -147,9 +147,14 @@
                 {%- else -%}
                     {# Display a warning telling the user we failed to build the docs #}
                     <div class="warning">
-                        docs.rs failed to build {{ details.name }}-{{ details.version }}<br>Please check the
+                        docs.rs failed to build {{ details.name }}-{{ details.version }}
+                        <br>
+                        Please check the
                         <a href="/crate/{{ details.name }}/{{ details.version }}/builds">build logs</a> and, if you believe this is
                         docs.rs' fault, <a href="https://github.com/rust-lang/docs.rs/issues/new/choose">open an issue</a>.
+                        <br>
+                        See <a href="/about/builds">Builds</a> for ideas on how to fix a failed build,
+                        or <a href="/about/metadata">Metadata</a> for how to configure docs.rs builds.
```

Comment:
I'd put this before the link to opening an issue

Reply:
How does this look?

![image](https://user-images.githubusercontent.com/23638587/89112051-ad09bc00-d42b-11ea-896c-e990747176b5.png)

Reply:
That's very nice!